eslint/babel-eslint-shared-fixtures/package.json

```diff
@@ -5,6 +5,7 @@
   "private": true,
   "dependencies": {
     "@babel/core": "workspace:^7.12.13",
+    "@babel/eslint-parser": "workspace:*",
```

Comment:
It is a private package serving as a hub of common test dependencies, so we can specify `workspace:*` here.